.github/workflows/erlang.yml

```diff
@@ -17,8 +17,8 @@ jobs:
 
     strategy:
       matrix:
-        otp: ['25', '26']
-        rebar: ['3.22']
+        otp: ['25', '26.2.5.1', '27']
```

Comment:
`26.2.5.1` is strange, here, but `26.2.5.2` crashes the tests (it came out yesterday). I'll install it locally and see if a bug report is in order.